### DISCUSSION TITLE
Camelidae : Sparse Mixture of Experts (8x34b in less than 80gb)

### DISCUSSION THREAD

**Discussion by logikstate** [ggml-org/llama.cpp#4955] [General]
Anybody seen this?

https://github.com/wuhy68/Parameter-Efficient-MoE

**Comment by Artefact2** [comment]
https://huggingface.co/serpdotai/sparsetral-16x7B-v2 also looks similar.

Would be nice to run those in llama.cpp in gguf format.

**Comment by Kimiko-AI** [comment]
Is there anyone working on this?
